Theory/Voicing.cs

```diff
@@ -0,0 +1,36 @@
+
+namespace Fretty.Theory;
+//TODO: Determine whether this should be a
+//      Struct or a CLass, given that essentially
+//      It's an array of restricted values
+//      I think since it's an array it should 
+//      stay as a class, but I could make
+//      the fret position a Struct (int between -1 and 24)
+//      This however could just be over-object-orienting
+
+public class Voicing
+{
+
+    // The name of the chord is dependent
+    // on the tuning, and so should be stored
+    // in a Tuning instance in a dictionary
+    // style, with {Chord: Voicing1, Voicing2, etc}
+    private int[] _positions;
+
+    public Voicing(int[] fretPositions, int[] positions)
```

Comment:
What's the intended use of a Voicing? ie) What is parameter `int[] positions`s role?